sql/src/Main.sk

```diff
@@ -119,9 +119,10 @@ untracked fun main(): void {
         .arg(Cli.StringArg("table-suffix").about("Optional suffix")),
     )
     .subcommand(
-      Cli.Command("set-user")
-        .about("Sets the user of the database")
-        .arg(Cli.StringArg("userID").positional().required().about("userID")),
+      Cli.Command("connected-as")
+        .about("Informs the database that it is a replicating peer")
+        .arg(Cli.StringArg("userId").required().about("user ID"))
```

Comment:
Could we give a few words more detail in this (and the following) "about" string?  As is these are a bit redundant with the arg name itself.